.github/workflows/wrapper-upgrade-execution.yml

```diff
@@ -9,6 +9,7 @@ jobs:
   upgrade_wrapper:
     name: Execution
     runs-on: ubuntu-latest
+    if: github.repository == 'gradle/android-cache-fix-gradle-plugin'
```

Comment:
This job fails in forked repos, see https://github.com/Goooler/android-cache-fix-gradle-plugin/actions/runs/5816176844

Reply:
Thanks for noting this. I would either remove this workflow from your fork or setup the correct credentials for this job in your fork.
Adding this check to this repository is noise in the codebase. We can't be worrying about forks.

Reply:
And I believe this job should not be executed in forks, it will send commits like 30480245dbfbf94c5dd227002f7a48a9c9bc498a.